jgrapht-core/src/test/java/org/jgrapht/AllTests.java

```diff
@@ -32,8 +30,8 @@
  * @author Barak Naveh
  */
 @RunWith(Suite.class)
-@Suite.SuiteClasses({ AllAlgTests.class, AllAlgUtilTests.class, AllGenerateTests.class,
-    AllGraphTests.class, AllTraverseTests.class, AllUtilTests.class })
+@Suite.SuiteClasses({ AllAlgTests.class, AllGraphTests.class, AllTraverseTests.class,
```

Comment:
By removing `AllGenerateTests` from `AllTests`, are the generator tests still executed? It seems that the new generator test classes aren't added any of the `All{}Tests.class` classes? Should they be added?